egs/tedlium/local/map_acronyms_transcripts.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# convert acronyms in swbd transcript to fisher convention
```

Comment:
Hi Dan, I have removed the file map_acronyms_transcripts.py and added the
timing info in the run.sh. Zhouyang has sent the detailed info on timing to
you. The pocolm doesn't take 11 hours.
Ke

On Wed, Jun 1, 2016 at 2:37 AM, Daniel Povey <notifications@github.com>
wrote:

> In egs/tedlium/local/map_acronyms_transcripts.py
> <https://github.com/danpovey/pocolm/pull/7#discussion_r65308568>:
>
> > @@ -0,0 +1,54 @@
> > +#!/usr/bin/env python
> > +# convert acronyms in swbd transcript to fisher convention
>
> You can remove the file map_acronyms_transcripts.py, I think the cantab
> data is already normalized.
>
> And please let us know the timings by writing them somewhere in the run.sh
> in a comment.
> I assume it didn't take you 11 hours like it did vince62s?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/danpovey/pocolm/pull/7/files/ddd18fd128e62478141a4ebb0ecca8fc9a086315#r65308568>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANVxSiR_r914oVbD-3ylRG9BU8HMqSkZks5qHSiNgaJpZM4IrOpF>
> .
>



-- 
Ke Li
Dept. of Electrical and Computer Engineering
Johns Hopkins University
Email: kli26@jhu.edu